Pod/Classes/WPMediaPickerViewController.m

```diff
@@ -602,14 +631,14 @@ - (UIView *)emptyView
     return _emptyView;
 }
 
-- (void)addEmptyViewToView
+/** Checks if the parentViewController is providing a custom empty ViewController to be added, if not, add a provided custom emptyView
```

Comment:
Thank you for adding the documentation. Can we also mention that `emptyViewControllerForMediaPickerController` takes precedence over `emptyViewForMediaPickerController` in the public documentation of these two functions in `WPMediaPickerViewController.h` 🙏

Reply:
Documentation updated here https://github.com/wordpress-mobile/MediaPicker-iOS/pull/399/commits/72ba07e4ceeca34ac4093fe4b6b11cdc8e5b8363 😄